src/main/java/seedu/address/model/assignment/Assignment.java

```diff
@@ -4,8 +4,11 @@
  * Represents an Assignment in the address book.
  */
 public class Assignment {
-    public final String assignmentName;
-    public final float score;
+    private String assignmentName;
```

Comment:
![image](https://github.com/user-attachments/assets/04e24050-adf2-4970-a9b5-cd037b8c41e9)

The code works however the UI seems to be a bit repetitive as there is `Ex01 `mentioned twice and `Ex02 `mentioned twice. Maybe a small tweak to the UI / `toString `method would work fine.

Reply:
![image](https://github.com/user-attachments/assets/1cc971d8-7306-4c3f-9911-8787a9148b8a)
ui has been updated to this